release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1610,6 +1610,12 @@ The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview featu
 
 * While creating a sample application, the *Developer* perspective creates multiple resources that depend on each other and must be completed in a specific order. Previously, the admission plug-in sometimes could not check one of these resources, preventing the *Developer* perspective from generating the sample application. This issue has been fixed. The code creates the resources in the required order, so creating a sample application is more stable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1933666[*BZ#1933666*])
 
+* Previously, the users were not able to create a knative service as a cluster local from the *Developer* perspective. This issue has now been fixed by changing the label required for making a service cluster local to `'networking.knative.dev/visibility': 'cluster-local'`. The users are now able to create a knative service as cluster local from the *Developer* perspective. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1978159[*BZ#1978159*])
```

Comment:
```suggestion
* Previously, users were not able to create a Knative service as a private service from the *Developer* perspective. This issue has now been fixed, by updating the label `'networking.knative.dev/visibility': 'cluster-local'`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1978159[*BZ#1978159*])
```
Some suggestion to make this one clearer - please always capitalize Knative also